### DISCUSSION TITLE
Running waybar with cava enabled gives only semicolons in the bar

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [Alexays/Waybar#2407] [Q&A]
Running waybar with cava enabled and with some audio playing only gives me semicolons where the module is. 
This is my config: [waybar config](https://gist.github.com/lokesh-krishna/90e20680a3151c6e55fb3d712b4a8cb4)

Screenshot of what this looks like:

![grim-14-08-2023-22-37-50](https://github.com/Alexays/Waybar/assets/20235646/68f88a82-241a-46c1-9029-cc65ad694b32)

It is the center module but all that's there are semicolons. There is no error or other output. 

I'm on Arch and running Hyprland 0.28.0. I have cava 0.9.0-1.

Please lemme know if there's any other relevant information that I can provide.

**Comment by smsteward** [comment]
I'm affected by this as well. I think the culprit is `cava` is latching onto the input monitor of the default audio device instead of the output monitor. I've tried the solutions in upstream `cava` issues 422 and and 451, but with no luck. I'm using `pipewire` and `wireplumber` to manage audio, so I think it's an issue with `cava` not playing nicely with `pipewire`.

**Comment by smsteward** [comment]
Ok, so I was _finally_ able to get it to work. Turns out it wasn't pipewire at all, just config errors.

Make sure you have `"format-icons" : ["▁", "▂", "▃", "▄", "▅", "▆", "▇", "█" ]` in your `cava` waybar config settings, and you either uncomment the `bar_delimiter` option in your cava config file, or set `"bar_delimiter": 0` in your waybar module setting.

**Reply by raffaelemancuso** [reply]
I have this problem, tried this and didn't work

**Reply by LukashonakV** [reply]
Please see my comments in #2801. But further discussion is better to proceed there

**Comment by LukashonakV** [comment]
In short, please try to increase the`delay` parameter to 4, 6, 8 seconds.  Restart waybar and wait for  this delay. Then try to play the music.  On the author's machine, the waybar starts much faster than the pipe wire server. This leads to early interaction with the audio server when it is not ready, and a silent module termination
